tests/test_orm_fields.py

```diff
@@ -27,6 +27,15 @@ class T:
     assert t.email == "x@x.com"
 
 
+def test_field_read_only():
+    class T:
+        name = f.Field("Name", read_only=True)
+
+    t = T()
+    t.name = "y"
+    assert t.name is None
```

Comment:
what does this test do? shouldnt setting the same rather throw an exception?

Reply:
Oh, I think this test is based on a prior iteration of the code. It should be updated.